dev/plugins/hu.elte.txtuml.project/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.ui.ide,
  org.eclipse.core.runtime,
  org.eclipse.jdt.core,
+ org.eclipse.cdt.core,
```

Comment:
Why is CDT listed here as a plugin dependency? It seems nothing changes without it.